distribution/src/scripts/micro-integrator.sh

```diff
@@ -294,7 +294,6 @@ do
     $JAVA_VER_BASED_OPTS \
     -Dcarbon.home="$CARBON_HOME" \
     -Dlogger.server.name="micro-integrator" \
-    -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager \
```

Comment:
This need to be added to bat as well, also doesn't removing this doesn't affect any logs? AFAIK, this is responsible for printing logs before OSGI starts.